nodejs.md

```diff
@@ -0,0 +1,41 @@
+# Node.js on Graviton
+
+Graviton is an excellent choice for running web applications with Node.js. There
+are a few considerations to be aware of to get the best performance.
+
+## Use Multiprocessing
+
+Node.JS is fundamentally single threaded and so on an instance with more than
+vCPU (which is most of them!), the node process will leave the CPU
+underutilized. There a few ways to improve this. 
```

Comment:
This isn't specific to Graviton. Across the getting started guide do we tend to point that out, or not?

Just trying to make sure we're consistent since this impacts perception of the migration.

Reply:
It matters *more* to Graviton, which is why I pointed it out.